runtime/frequency/src/lib.rs

```diff
@@ -429,7 +429,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: spec_name!("frequency"),
 	impl_name: create_runtime_str!("frequency"),
 	authoring_version: 1,
-	spec_version: 28,
+	spec_version: 29,
```

Comment:
I wouldn't have expected this to be needed.

Reply:
The CI broke since it is going in main

Reply:
https://github.com/LibertyDSNP/frequency/actions/runs/4733631145